modules/intermodal/include/motis/intermodal/intermodal.h

```diff
@@ -26,6 +26,7 @@ struct intermodal : public motis::module::module {
 
   std::string router_{"routing"};
   bool revise_{false};
+  int timeout_ = 0;
```

Comment:
```suggestion
  unsigned timeout_{0};
```

As negative values don't make sense here, maybe `unsigned` and please use the `var_{..}` style (just for consistency - both styles work here).